### DISCUSSION TITLE
好きなスポーツを教えてください！Please tell me your favorite sports!!

### DISCUSSION THREAD

**Discussion by daiki-may** [NAIST-SE/AttractingContributors#149] [General]
https://github.com/daiki-may/AttractingContributors

**Comment by nagata-atsuki** [comment]
added!!!
Please help me!
https://github.com/nagata-atsuki/AttractingContributors-Athlete

**Comment by MatsumuraTaiki** [comment]
Added, please help me too
https://github.com/MatsumuraTaiki/AttractingContributors-Actors/edit/main/README.md

**Comment by veekay7** [comment]
Added to your repo.
Please help.
https://github.com/VantelinLuca/AttractingContributors-WishCountry
